frontend/src/navbar/UserStatusContextMenu.tsx

```diff
@@ -31,7 +31,7 @@ export const UserStatusContextMenu = ({ x, y, userName, onLoggedOut, onClose })
       </div>
       <Separator />
       <a href="https://ocpmodeler.com/doc/welcome_help_center.html" rel="noopener noreferrer" target="_blank">
-        <Entry icon={<Help title="Help" />} label="Help" data-testid="help" />
+        <Entry icon={<Help titleAccess="Help" />} label="Help" data-testid="help" />
```

Comment:
Has the Help component been migrated to SvgIcon? If we went this way, we should probably just replace it with the regular MaterialUI help icon and probably remove the titleAccess here. It does not provide any meaningful information

Reply:
Done